eslint-plugin/lib/rules/limit-db-query-results.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (C) 2023 Green Code Initiative
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+"use strict";
+
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  meta: {
+    type: "suggestion",
+    docs: {
+      description: "Warn about not limiting the number of returns for a query",
+      category: "eco-design",
+      recommended: "warn",
+    },
+    messages: {
+      LimitTheNumberOfReturns:
+        "Try and limit the number of data returned for a single query (by using the LIMIT keyword for example)",
+    },
+    schema: [],
+  },
+  create(context) {
+    return {
+      Literal: function (node) {
+        if (typeof node.value == "string") {
+          const query = node.value.toUpperCase();
+          if (
+            query.includes("SELECT") &&
+            query.includes("FROM") &&
+            !(
+              query.includes("LIMIT") ||
+              query.includes("TOP") ||
+              query.includes("ROW_NUMBER") ||
+              query.includes("FETCH FIRST")
```

Comment:
Maybe store the list of keywords in a constant above?
The goal is to simplify the addition of keywords that limit number of returns in the future :+1: